integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java

```diff
@@ -0,0 +1,836 @@
+import static oracle.weblogic.kubernetes.utils.FileUtils.generateFileFromTemplate;
+import static oracle.weblogic.kubernetes.utils.FmwUtils.verifyDomainReady;
+import static oracle.weblogic.kubernetes.utils.ImageUtils.createMiiImageAndVerify;
+import static oracle.weblogic.kubernetes.utils.ImageUtils.createOcirRepoSecret;
+import static oracle.weblogic.kubernetes.utils.ImageUtils.createSecretForBaseImages;
+import static oracle.weblogic.kubernetes.utils.ImageUtils.dockerLoginAndPushImageToRegistry;
+import static oracle.weblogic.kubernetes.utils.IstioUtils.deployHttpIstioGatewayAndVirtualservice;
+import static oracle.weblogic.kubernetes.utils.IstioUtils.deployIstioDestinationRule;
+import static oracle.weblogic.kubernetes.utils.IstioUtils.getIstioHttpIngressPort;
+import static oracle.weblogic.kubernetes.utils.OKDUtils.createRouteForOKD;
+import static oracle.weblogic.kubernetes.utils.OperatorUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.PatchDomainUtils.patchDomainResourceServerStartPolicy;
+import static oracle.weblogic.kubernetes.utils.PersistentVolumeUtils.createPV;
+import static oracle.weblogic.kubernetes.utils.PersistentVolumeUtils.createPVC;
+import static oracle.weblogic.kubernetes.utils.PodUtils.checkPodDeleted;
+import static oracle.weblogic.kubernetes.utils.PodUtils.checkPodDoesNotExist;
+import static oracle.weblogic.kubernetes.utils.PodUtils.getExternalServicePodName;
+import static oracle.weblogic.kubernetes.utils.SecretUtils.createOpsswalletpasswordSecret;
+import static oracle.weblogic.kubernetes.utils.SecretUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test to a create Istio enabled FMW model in image domain and WebLogic domain using Oracle "
+    + "database created using Oracle Database Operator")
+@IntegrationTest
+class ItIstioDBOperator {
+
+  private static String dbNamespace = null;
+  private static String opNamespace = null;
+  private static String fmwDomainNamespace = null;
+  private static String wlsDomainNamespace = null;
+  private static String fmwMiiImage = null;
+
+  private static final String RCUSCHEMAPREFIX = "FMWDOMAINMII";
+  private static final String RCUSYSPASSWORD = "Oradoc_db1";
+  private static final String RCUSCHEMAPASSWORD = "Oradoc_db1";
+  private static final String modelFile = "model-singleclusterdomain-sampleapp-jrf.yaml";
+
+  private static String dbUrl = null;
+  private static String dbName = "my-oracle-sidb";
+  private static LoggingFacade logger = null;
+
+  private String fmwDomainUid = "fmwdomain-mii-db";
```

Comment:
I want the domainuid to be unique across all the class for better triaging prospective.
$ grep fmwdomain-mii-db *.java
ItDBOperator.java:  private String fmwDomainUid = "fmwdomain-mii-db";

Can u make the domaiuid to be unique such as 'fmwdomain-istio-mii-db"

Reply:
fixed